src/components/MarkdownWrapper/MarkdownWrapper.style.tsx

```diff
@@ -51,9 +52,18 @@ export const MarkdownWrapperStyled = styled.div.attrs<MarkdownWrapperStyledProps
   }
 `;
 
-export const ButtonLink = styled.a.attrs((props) => ({
+type ExtraEventType = { trackEvent?: string };
+export const ButtonLink = styled.a.attrs((props: Record<string, any>) => ({
   ...props,
   className: `fr-btn ${props.className || ''}`,
+  ...((props as ExtraEventType)?.trackEvent && {
+    onClick: () => {
+      matomoEvent(markupData.markdownEvent.matomoEvent, [
+        (props as any).trackEvent ||
```

Comment:
If you're here you have a trackEvent so the || is useless

Reply:
The trackEvent can also be an empty string.

Reply:
then (props as ExtraEventType)?.trackEvent return false

Reply:
Ho, sure ! My bad.